packages/embark-deployment/src/contract_deployer.js

```diff
@@ -96,6 +96,10 @@ class ContractDeployer {
 
     async.waterfall([
       function checkContractBytesize(next) {
+        if (contract.code === undefined) {
```

Comment:
Could this be `!contract.code` instead, for `null` or whatever?

Reply:
Updated!